### PR TITLE
[BE] Validate a job

### PR DESCRIPTION
**Pull request by nspc97**
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

### CODE REVIEW

File: src/Controller/ActivityController.php [hunk 1]
```diff
@@ -1536,4 +1536,103 @@ public function getActivitiesForValidation(ActivityRepository $activityRepositor
         );
         return new JsonResponse($json, 200, [], true);
     }
+
+    /**
+     * Validate an applicant.
```
Comment (MihaiDruta) [inline src/Controller/ActivityController.php]: Should be: validate a Job

**Review by MihaiDruta — APPROVED** [review]

**Review by SpartakusMd — APPROVED** [review]

**Review by SpartakusMd — CHANGES_REQUESTED** [review]
Add tests for validation:
- Successful test
- one test per each failing case

File: src/DataFixtures/UserFixtures.php [hunk 2]
```diff
@@ -22,13 +25,25 @@ class UserFixtures extends Fixture
...
+     * @param UserRepository $userRepository
+     * @throws ORMException
+     * @throws OptimisticLockException
+     */
+    public function __construct(UserPasswordEncoderInterface $encoder, UserRepository $userRepository)
     {
         $this->encoder = $encoder;
+        $users = $userRepository->findAll();
+        foreach ($users as $user) {
+            $userRepository->unassignProjectManager($user);
```
Comment (SpartakusMd) [inline src/DataFixtures/UserFixtures.php]: Why are you doing this in fixtures? Everything from database is removed at each load of fixtures.

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

**Comment by SpartakusMd** [comment]
Travis build is failing. Check why and fix the issue.

**Review by SpartakusMd — APPROVED** [review]

**Pull request merged**